packages/web/package.json

```diff
@@ -17,7 +17,8 @@
     "graphql": "16.2.0",
     "proptypes": "1.1.0",
     "react-helmet-async": "1.2.2",
-    "react-hot-toast": "2.1.1"
+    "react-hot-toast": "2.1.1",
+    "stacktracey": "2.1.7"
```

Comment:
[This lib](https://github.com/xpl/stacktracey) is pretty essential at getting structured data out of stack traces - they're a lot to work with (source-maps, location extraction etc) .